src/apps/setup.ts

```diff
@@ -39,14 +39,22 @@ export const setupAppFactory = (port: number | undefined) =>
       const { code } = req.query;
       const response = await setup.createAppFromCode(code);
 
-      restartGlitchApp(app);
+      // If using glitch, restart the app
+      if (process.env.PROJECT_DOMAIN) {
+        exec("refresh", (error) => {
+          if (error) {
+            app.log.error(error);
+          }
+        });
+      }
```

Comment:
I wonder, when we are not on glitch, we could at least log out an instruction to restart the server at this point. But we can do this in a follow up PR, I'll make an issue as a reminder